src/routes/legal/privacy-policy.md

```diff
@@ -19,7 +19,7 @@ If you want the data we've collected on you to be sent, please contact us on Mat
 
 We have Plausible analytics installed on our website (the one you are on right now!), and our [Blog](https://blog.projectsegfau.lt), which means we can just easily see what part of our site you're on, what country you are from, what platform you are from, potentially a referrer and all those get clumped up together.
 
-**Plausible analytics does not collect any PII(Personally Identifiable Information)**
+**Plausible Analytics is completely anonymous. We can't exactly figure out who you are through Plausible. It is also GDPR compliant.**
```

Comment:
could you please change plausible analytics to plural? it makes more sense since analytics is plural as well

Reply:
Plausibles analytics?

Reply:
i mean the `Plausible analytics is`, change it to `are`

Reply:
Doesn't really sound correct to me

Reply:
yeah, seeing their site, they use the singular form as well, marking this as resolved